storm-client/src/jvm/org/apache/storm/dependency/DependencyUploader.java

```diff
@@ -44,9 +51,12 @@
 
     private final Map<String, Object> conf;
     private ClientBlobStore blobStore;
+    private static final int DEFAULT_CHUCK_SIZE = 1024 * 1024;
```

Comment:
I think this should go in https://github.com/apache/storm/blob/master/conf/defaults.yaml so it's obvious to users what the default value is.

Reply:
thanks for the heads up !